homeworks/irina.vlasova_IrinaBondarenko/homework_6/horse.js

```diff
@@ -0,0 +1,69 @@
+class Horse {
+    constructor(name, breed) {
+        this.name = name;
+        this.breed = breed;
+    }
+}
+class Racer extends Horse {
+    constructor(name, breed) {
+        super(name, breed);
+        this.distance = 0;
+        this.speed = 0;
+        this.setSpeed();
+    }
+
+    setSpeed() {
+        this.speed = Math.floor(Math.random() * 5 + 10);
```

Comment:
multiplying by 6 maybe more suitable here, because it will include 15 in the set, but in general it is correct